README.md

```diff
@@ -1,9 +1,102 @@
-Libero search
-=============
+# Libero Search
+The purpose of this project is to provide the ability to search indexed data via the
+`/search` endpoint.
```

Comment:
Better to describe it as an implementation the Libero search API.

Reply:
updated.